### DISCUSSION TITLE
Swagger access in self-hosted

### DISCUSSION THREAD

**Discussion by andreyddk** [Infisical/infisical#1314] [Q&A]
Hi! How to get access to swagger? I was deployed the infisical at my k8s and he is works pretty good, but I have some problems with swagger access.

My host at http://10.1.0.1:30601/api-docs/
And I see the errors with loading of swagger bundle https://10.1.0.1:30601/api-docs/swagger-ui-bundle.js

`ERR_SSL_PROTOCOL_ERROR`

How can I resolve it?

**Comment by maidul98** [comment]
Infisical cloud uses the same image and you can see it load up here https://app.infisical.com/api-docs/. Do you see any errors logs 
in the backend?

**Reply by andreyddk** [reply]
No, I did not see any errors.
